lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json

```diff
@@ -8,6 +8,6 @@
         "objectClass": ["top", "groupofNames"],
         "cn": "#cn",
         "description": "#description",
-        "member": [#members]
+        "member": ["#members"]
     }
```

Comment:
Will revert, created https://github.com/lightblue-platform/lightblue-ldap/issues/65 to fix